.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,26 @@
+name: Update python dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "5 6 * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: "opensafely-core/setup-action@v1"
+      with:
+        python-version: "3.12"
+        install-just: true
+
+    - uses: actions/create-github-app-token@v1
+      id: generate-token
+      with:
+        app-id: 1031449  # opensafely-core Create PR app
+        private-key: ${{ secrets.CREATE_PR_APP_PRIVATE_KEY }}
```

Comment:
@rebkwok: The app is now installed in this organization.

Does this need a new private key creating and adding to the app, for this organization? Or do we use the existing key (which I don't think I have access to)?

Reply:
Sorry, missed this comment. I think this is done now, but no, one it just needs the app's private key